### PR TITLE
Make validations for login page

### PR DESCRIPTION
**Pull request by avolchivschi**

### CODE REVIEW

File: intellst-front/src/views/Login.vue [hunk 1]
```diff
@@ -50,15 +66,36 @@ import { mapActions } from "vuex";
 
 export default {
   data: () => ({
+    status: null,
     user: {
       username: "",
       password: "",
     },
+    show1: false,
```
Comment (andreikojusko) [inline intellst-front/src/views/Login.vue]: please use "talking" variable name instead of show1

File: intellst-front/src/views/Login.vue [hunk 2]
```diff
@@ -50,15 +66,36 @@ import { mapActions } from "vuex";
...
+      return this.status === 401;
+    },
+  },
   methods: {
     ...mapActions("login", ["signInAction"]),
     loginUser() {
       this.signInAction(this.user);
+      setTimeout(() => {
+        this.status = 401;
+        console.log(this.show);
```
Comment (andreikojusko) [inline intellst-front/src/views/Login.vue]: is this console.log needed?

**Review by andreikojusko — CHANGES_REQUESTED** [review]
feel free to merge after resolving

File: intellst-front/src/views/Login.vue (hunk 1 above)
Comment (avolchivschi) [reply intellst-front/src/views/Login.vue]: Done

File: intellst-front/src/views/Login.vue (hunk 2 above)
Comment (avolchivschi) [reply intellst-front/src/views/Login.vue]: Done

**Pull request merged**